index.html

```diff
@@ -2,245 +2,234 @@
-        <h1 class="h1-main">
-          Pre-order titles by console
-        </h1>
-    </div>
-    <div class="col">
-    </div>
-  </div>
-
-  <div class="flex-grid-thirds">
-    <div class="col">
-      <div id="ps-box">
-      <img id="ps-logo" src="Assets/Desktop/Preorder_console/PlayStation_logo.png">
-       <p class="p-sub">
-          Playstation
-            <img id ="chevron_down" src="Assets/Desktop/Preorder_console/chevron_right.png">
-        </p>
+<body>
+  <div class="wrapper">
+    <!--NAV-->
+    <nav class="header-navigation">
+      <div class="header-navigation--container">
+        <div class="header-navigation--logo">
+          <img src="Assets/Desktop/header/logos/Best Buy.png">
+          <img id="E3" src="Assets/Desktop/header/E3.png">
+        </div>
+        <div class="language">
+          <img id="lang-fr" src="Assets/Desktop/header/buttons/text/xsmall/white + right icon.png">
+        </div>
       </div>
-    </div>
-    <div class="col">
-      <div id="xbox-box">
-        <img id="xbox-logo"  src="Assets/Desktop/Preorder_console/xbox_logo.png">
-        <p class="p-sub">
-          X-Box
-            <img id ="chevron_down" src="Assets/Desktop/Preorder_console/chevron_right.png">
-        </p>
+    </nav>
+
+    <!--HERO-->
+    <div class="hero">
+      <div class="hero--container">
+        <div class="hero--left-thumb">
+          <img src="Assets/Desktop/hero/left-thumb-up.png">
+        </div>
+        <div class="hero--lockup">
+          <div class="lockup-image-container">
+            <img src="Assets/Desktop/hero/MainHero_Lockup.svg">
+          </div>
+          <div class="hero--lockup-content">
+            <p>Come back on June 8 for our exclusive E3 offer.</p>
+          </div>
+        </div>
+        <div class="hero--right-thumb">
+          <img src="Assets/Desktop/hero/right-thumb-group.png">
+        </div>
       </div>
-    </div>
-      <div class="col">
-      <div id="nintendo-box">
-        <img id="nintendo-logo"  src="Assets/Desktop/Preorder_console/Nintendo_Red_Logo.png">
-        <p class="p-sub">
-          Nintendo
-            <img id ="chevron_down" src="Assets/Desktop/Preorder_console/chevron_right.png">
-        </p>      
+      <div class="notification-bar">
+          <p class="--sub-heading">Follow us on Twitter
+              <b>@BBYC_Gamers</b> for the latest news, hottest offers, and exclusive contests.</p>
       </div>
     </div>
-  </div>
-
 
-
-<!--PREORDER BY LATEST-->
-  <div class="flex-grid-thirds">
-    <div class="col">
-    </div>
-    <div class="col">
-      <h1 class="h1-main">
-        Pre-order latest games
-      </h1>
-    </div>
-    <div class="col">
+    <!--PREORDER BY CONSOLE-->
+    <div class="preorder-by-console page-section">
+      <div class="page-section--heading">
+        <h1 class="--heading-text">Pre-order titles by console</h1>
+      </div>
+      <div class="page-section--content">
+        <div class="grid-container">
```

Comment:
created a shared `grid-container` class
this is responsible for handling the 3-column grids and 2 column grids universally

the rule is to make sure that each children uses "grid-item" class